add-ons/grafana/README.md

```diff
@@ -0,0 +1,39 @@
+# Grafana Add-on
+An addon that will deploy grafana with hawkular and prometheus datasources.
+It requires to connect to hawkular-metrics, provided by management-infra addon.
+
+## Deploy grafana
+To deploy grafana do:
+
+```
+minishift addon apply grafana --addon-env namespace=grafana
+```
+
+_NOTE_: You should provide the namespace where it will be installed with the addon-env namespace.
+
+## Use grafana
+grafana will be available at:
+
+```
+$ minishift openshift service hawkular-grafana -n <namespace>
+```
+
+Add a datasource with:
+* Type: Hawkular
+* Url: the route given on "minishift openshift service hawkular-metrics -n openshift-infra" ended with /hawkular/metrics. Ex: http://hawkular-metrics-openshift-infra.192.168.99.100.nip.io/hawkular/metrics
```

Comment:
I am getting 
```
$ minishift openshift service hawkular-metrics -n openshift-infra
No services defined in namespace 'openshift-infra'
```

Did I miss anything?

Reply:
See the following review.

Reply:
I am still getting above error after doing
- Fresh install of minishift
- Apply management-infra add-on
- Apply grafana add-on

Any idea?

Reply:
Yes. Metrics should be enabled when started minishift. I should add this too.

Reply:
@gordillo-ramon-redhat ah ok. Restarting now.

Reply:
Got it now
```
$ minishift openshift service hawkular-metrics -n openshift-infra
|-----------------|------------------|----------|---------------------------------------------------------------|--------|
|    NAMESPACE    |       NAME       | NODEPORT |                           ROUTE-URL                           | WEIGHT |
|-----------------|------------------|----------|---------------------------------------------------------------|--------|
| openshift-infra | hawkular-metrics |          | http://hawkular-metrics-openshift-infra.192.168.42.124.nip.io |        |
|-----------------|------------------|----------|---------------------------------------------------------------|--------|
```

Reply:
However, some issue with my metrics URL. Still `Application is not available` page.

Reply:
I think you have hit a bug in minishift. I have double checked, and I get the same route as you using minishift. With minishift:

```
$ minishift openshift service hawkular-metrics -n openshift-infra
|-----------------|------------------|----------|---------------------------------------------------------------|--------|
|    NAMESPACE    |       NAME       | NODEPORT |                           ROUTE-URL                           | WEIGHT |
|-----------------|------------------|----------|---------------------------------------------------------------|--------|
| openshift-infra | hawkular-metrics |          | http://hawkular-metrics-openshift-infra.192.168.99.100.nip.io |        |
|-----------------|------------------|----------|---------------------------------------------------------------|--------|
```

but with oc:

```
$ oc get route hawkular-metrics -n openshift-infra -o wide 
NAME               HOST/PORT                                                PATH      SERVICES           PORT      TERMINATION   WILDCARD
hawkular-metrics   hawkular-metrics-openshift-infra.192.168.99.100.nip.io             hawkular-metrics   <all>     reencrypt     None

```
I used oc for my tests, and I thought with minishift it will be the same. Will update the docs though.